recipes/naiad-commons/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "naiad-commons" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://gitlab.ifremer.fr/naiad/naiad-commons.git
```

Comment:
```suggestion
  url: https://gitlab.ifremer.fr/naiad/naiad-commons/-/archive/{{ version }}/naiad-commons-{{ version }}.tar.gz
```

Reply:
With the use of poetry-dynamic-versioning the url does not work it seems to need the .git url. Is it really not possible to use the git_url ?

Reply:
See https://github.com/conda-forge/staged-recipes/pull/21655#discussion_r1070989128

PS: For simplicity let's keep the discussion there too and, when we resolve, we can apply the solution here as well.